lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -37,7 +38,11 @@ def get_glyphs_subset(fontfile, characters):
     options = subset.Options(glyph_names=True, recommended_glyphs=True)
 
     # prevent subsetting FontForge Timestamp and other tables
-    options.drop_tables += ['FFTM', 'PfEd']
+    options.drop_tables += ['FFTM', 'PfEd', 'BDF']
+
+    # if fontfile is a ttc, specify font number
+    if os.path.splitext(fontfile)[1] == ".ttc":
```

Comment:
Probably don't need to do a whole parsing:
```suggestion
    if fontfile.endswith(".ttc"):
```

Reply:
right, reasonable.. updating